src/garage/tf/algos/npo.py

```diff
@@ -105,13 +107,15 @@ def __init__(self,
                  stop_entropy_gradient=False,
                  entropy_method='no_entropy',
                  flatten_input=True,
+                 fit_baseline_first=False,
                  name='NPO'):
         self._name = name
         self._name_scope = tf.name_scope(self._name)
         self._use_softplus_entropy = use_softplus_entropy
         self._use_neg_logli_entropy = use_neg_logli_entropy
         self._stop_entropy_gradient = stop_entropy_gradient
         self._pg_loss = pg_loss
+        self._will_fit_baseline_first = fit_baseline_first
```

Comment:
please keep member variables the same as constructor args, so they're easy to track